LocalyticsInbox/src/InboxDetailView.js

```diff
@@ -1,17 +1,24 @@
 import React from 'react';
 
 import { LLWebView } from 'localytics-react-native';
+import { StyleSheet } from 'react-native';
 
 export default class InboxDetailView extends React.Component {
   static navigationOptions = {
     title: 'Inbox Detail View',
   };
 
   render() {
-    const { navigation: { state: { params: { campaignId } } } } = this.props;
+    const { navigation: { state: { params: { creative } } } } = this.props;
```

Comment:
creative is just the variable name so it should not matter what it is called. Is it the style that was causing the issue?
If it is the sale  do we need a docs update?

Reply:
The variable name was a mismatch though with what it was delivered as - causing a null value. The variable name passed into the navigation is `creative`.  They just have to match. 

The style fixes help display the webview, otherwise it can shrink depending on the html content.